config/initializers/monkeypatch_vcr.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
```

Comment:
VCR is not loaded in the production system. But I guess if VCR isn't loaded, this creates a useless module VCR that doesn't do anything, so I guess this should work. If that's wrong let me know...

Reply:
Actually, just to be safe, let's add a check for that